pkg/cmd/experimental/syncgroups/groupdetector/groupdetector.go

```diff
@@ -76,13 +80,13 @@ type CompoundDetector struct {
 }
 
 func (l *CompoundDetector) Exists(ldapGrouUID string) (bool, error) {
-	conclusion := false
+	conclusion := true
```

Comment:
I don't think we should allow a `CompoundDetector` that has no constituent detectors. What would that be useful for?